internal/sdkgen/sdkgen.go

```diff
@@ -94,6 +102,30 @@ func Generate(ctx context.Context, customerID, workspaceID, lang, schemaPath, he
 	return nil
 }
 
+func processDocsCfg(outDir string) generate.GeneratorOptions {
```

Comment:
Can you do this inside the generator? Doing it here feels like a workaround for handling it natively

Reply:
We could. It's going to have to be really early since we need this languages when generating config. I can add it to that other PR and show you where it would need to live.

Reply:
https://github.com/speakeasy-api/openapi-generation/pull/961/files

Updated this so you can see what it would look like